loadtest/user/userentity/user.go

```diff
@@ -60,21 +61,24 @@ func New(store store.MutableUserStore, id int, config Config) *UserEntity {
 	ue.store = store
 	ue.closing = make(chan struct{})
 	ue.closed = make(chan struct{})
+	ue.wsErrorChan = make(chan error, 1)
 	return &ue
 }
 
 // Connect creates a websocket connection to the server and starts listening for messages.
-func (ue *UserEntity) Connect() error {
+func (ue *UserEntity) Connect() chan error {
```

Comment:
I'd like to be explicit with the direction of the channel here to make sure it's *readonly* for the controller.